bu-ist/responsive-framework
# Migrate 1x/update dependencies june18

This automatically fixes known security issues in NPM using `npm audit` and removes any unused packages from Responsive. Removed packages include `grunt-bowercopy`.

**To review/test this change:**

- Download this branch from https://github.com/bu-ist/responsive-framework/tree/update-dependencies-june18
- cd to the downloaded folder
- run `npm install` and confirm there are no errors

This pull request has been migrated from https://github.com/bu-ist/responsive-framework-1x/pull/407 as part of our history migration effort.
